packages/app/src/client/integrations/k8sMetrics/Show/dashboards/index.ts

```diff
@@ -27,7 +27,7 @@ type DashboardProps = {
 export function makePrometheusDashboardRequests({
   integrationId,
   folderId
-}: DashboardProps): object[] {
```

Comment:
Small clean up. It's better to have no typing than bad generic typing, as non-explicit typing can often be inferred by TS.